.github/workflows/integration-opensuse-leap.yml

```diff
@@ -14,4 +14,4 @@ jobs:
           repository: openSUSE/salt-toaster
           path: salt-toaster
       - name: Integration tests
-        run: cd $GITHUB_WORKSPACE/salt-toaster && make saltstack.integration VERSION=opensuse151 FLAVOR=devel SALT_REPO=$GITHUB_WORKSPACE/salt
+        run: cd $GITHUB_WORKSPACE/salt-toaster && make saltstack.integration VERSION=opensuse152 FLAVOR=devel SALT_REPO=$GITHUB_WORKSPACE/salt
```

Comment:
Can we use Leap 15.3? 15.2 is EOL end of november (https://en.opensuse.org/Lifetime#openSUSE_Leap)